app/src/main/java/com/example/una/MapsViewFragment.java

```diff
@@ -0,0 +1,6 @@
+package com.example.una;
+
+import android.support.v4.app.Fragment;
+
+public class FragmentMapsView extends Fragment {
```

Comment:
Does this compile? Pretty sure the name of the public class should match the name of the file